packit_service/worker/helpers/build/copr_build.py

```diff
@@ -917,7 +917,11 @@ def create_or_update_copr_project(self) -> str:
                 f" `{owner}/{self.job_project}` "
                 "specified in the config with the following error:\n"
                 f"```\n{str(ex)}\n```\n---\n"
-                "Please check your configuration for:\n\n"
+                "If the reported error is a 5XX Copr server error"
+                "check the Copr status page: https://copr.fedorainfracloud.org/status/stats/, "
+                "or ask for help in Fedora Build System matrix channel "
+                "https://matrix.to/#/#buildsys:fedoraproject.org.\n"
+                "Otherwise, please check your configuration for:\n\n"
```

Comment:
I think all the info is already in the exception text (`ex` above), so what about just something like:
```suggestion
                "Unless the HTTP status code above is >= 500, please check your configuration for:\n\n"
```

Reply:
Oh right 🙈🙏